packages/esm-patient-orders-app/src/index.ts

```diff
@@ -56,6 +56,10 @@ export const ordersDashboardLink =
 
 export const ordersDashboard = getSyncLifecycle(OrdersSummary, options);
 export const labResult = getAsyncLifecycle(() => import('./lab-results/lab-result.component'), options);
+export const editLabResultsModal = getAsyncLifecycle(
```

Comment:
The "edit lab results modal" should be defined in the test app and not the orders app.